aten/src/ATen/native/mkldnn/xpu/Attention.cpp

```diff
@@ -87,8 +87,8 @@ sdp::SDPBackend select_sdp_backend_xpu(sdp::sdp_params const& kernel_params) {
   // Get ideal kernel ordering
   const std::array<sdp::SDPBackend, 3> priority_order{
       sdp::SDPBackend::overrideable,
-      sdp::SDPBackend::math,
       sdp::SDPBackend::flash_attention,
+      sdp::SDPBackend::math,
```

Comment:
I guess we currently don't support set_priority=True since our `priority_order` is a constant value.